docs/storage/storage.md

```diff
@@ -98,3 +99,57 @@ comment the next property:
   <description>notebook persistence layer implementation</description>
 </property>
 ```   
+
+</br>
+#### Notebook Storage  in Azure <a name="Azure"></a>
+
+Using `AzureNotebookRepo` you can connect your Zeppelin with your Azure account for notebook storage.
+
+</br>
+
+First of all, input your `AccountName`, `AccountKey`, and `Share Name` in the file **zeppelin-site.xml** by commenting out and completing the next properties:
+```
```

Comment:
@khalidhuseynov Currently this docs is rendered like this
![screen shot 2016-05-24 at 3 05 25 pm](https://cloud.githubusercontent.com/assets/10060731/15493750/0b77d712-21c1-11e6-9a56-0340556e3b44.png)

So, please add a new line between the each general sentences and code blocks.